Server/Network/MessageProcessor.cs

```diff
@@ -390,11 +390,11 @@ internal static void ProcessData(Client client, string[] parse)
                                 {
                                     PlayerManager.DeleteCharacter(dbConnection, client.Player.AccountName, charNum);
                                     Messenger.SendChars(dbConnection, client);
-                                    Messenger.PlainMsg(client, "Character has been deleted!", Enums.PlainMsgType.Chars);
+                                    Messenger.PlainMsg(client, "The character has been sent back from the Pokémon world, never to return.", Enums.PlainMsgType.Chars);
```

Comment:
Yeah, but it'll have lost all its progress. It's like having CharacterName I and CharacterName II.